### DISCUSSION TITLE
Clarity on some language in the IIIF section of the API docs

### DISCUSSION THREAD

**Discussion by jessamynwest** [harvardartmuseums/api-docs#16] [Q&A]
I noted the language in the IIIF section says "Please note that URLs for images of objects that have rights restrictions are excluded for most API users. This means images for many 20th and 21st century works of art will not be available to you at the present time." 

Does this mean that the images that are pullable from the Harvard Art Museum do not have rights restrictions, i.e. are public domain? I ask because I've found quite a few that have more restrictive language attached to the actual items, for example: "The Harvard Art Museums encourage the use of images found on this website for personal, noncommercial use, including educational and scholarly purposes. To request a higher resolution file of this image, please submit an online request."

I am not trying to do anything with these images that isn't allowed by HAM, but I am unclear if certain things only allowable for public domain or CC-BY images such as putting the images on Wikipedia, is allowed for these images? Here is one example image.

https://harvardartmuseums.org/collections/object/346527

**Comment by jeffsteward** [comment]
The vast majority of image URLs available to most users of the API fall in to the public domain. At present it's near impossible to use the API to sort out which have complications, We are deep in the process of sorting this out in our internal database. These updates will soon be translated to standard language on every record and flags to make it easy to filter by public domain status. Coming soon. 

The following statement appears on every object page on our public website: "The Harvard Art Museums encourage the use of images found on this website for personal, noncommercial use, including educational and scholarly purposes. To request a higher resolution file of this image, please submit an online request." It's main purpose is to signal that we have higher resolution versions of the images found online. Unfortunately these files are not directly accessible from the web because of the way we archive them. The only way to get them is to submit a request through our [online request form](https://harvardartmuseums.org/image-licensing).
